src/components/Slider/Slider.tsx

```diff
@@ -16,7 +16,7 @@ export const Slider: React.FC = () => {
   );
 
   useEffect(() => {
-    dispatch(init());
+    dispatch(fetchProducts(1));
```

Comment:
it's better to define default property value (1) in the `fetchProducts` function instead of passing it from the place where it's used